packages/fast-usdc/src/cli/transfer.js

```diff
@@ -0,0 +1,92 @@
+/* global globalThis */
+
+import { makeVStorage } from '@agoric/client-utils';
+import { depositForBurn, makeProvider } from '../util/cctp.js';
+import {
+  makeSigner,
+  queryForwardingAccount,
+  registerFwdAccount,
+} from '../util/noble.js';
+import { queryFastUSDCLocalChainAccount } from '../util/agoric.js';
+
+/** @typedef {import('../util/file').file} file */
+/** @typedef {import('@agoric/client-utils').VStorage} VStorage */
+/** @typedef {import('@cosmjs/stargate').SigningStargateClient} SigningStargateClient */
+/** @typedef {import('ethers').ethers.JsonRpcProvider} ethProvider */
+
+const transfer = async (
+  /** @type {file} */ configFile,
+  /** @type {string} */ amount,
+  /** @type {string} */ destination,
```

Comment:
The rest of agoric-sdk uses `@param`. This is more concise in a way, so I don't mind too much. It's also more like typescript, which we seem to be headed toward. So maybe it's better.

Reply:
LGTM. We might have the whole package on `.ts` soon: https://github.com/Agoric/agoric-sdk/pull/10480